specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json

```diff
@@ -210,7 +210,7 @@
           "readOnly": true,
           "x-ms-enum": {
             "name": "RestorePointType",
-            "modelAsString": true
+            "modelAsString": false
```

Comment:
why? Note that this is highly discourages since it makes any change to this set a breaking change.

Reply:
This is to comply with 2014 version and for backward compatibility. Because string is a breaking change and enum is what sdk looks like today in sdk.

Reply:
@jaredmoo